vendor/k8s.io/kube-openapi/pkg/generators/openapi.go

```diff
@@ -411,6 +412,19 @@ func (g openAPITypeWriter) generate(t *types.Type) error {
 	return nil
 }
 
+func getOpenAPITypeFormat(t *types.Type) (string, string) {
+	schemaTypeFormatTag := types.ExtractCommentTags("+", t.CommentLines)[tagSchemaTypeFormat]
+	for _, val := range schemaTypeFormatTag {
```

Comment:
there should be only one of these tags. If there is more than one, the generator should fail with an error message.

Reply:
Fixed.